### PR TITLE
 make Mr. and Mrs. translatable

### PR DESCRIPTION
**Pull request by fabtho**
make gender Mr. and Mrs. translateabel. See diff

### CODE REVIEW

File: Resources/Private/Language/locallang.xml [hunk 1]
```diff
@@ -22,6 +22,8 @@
             <label index="flexforms.from_name">Name of sender used in confirmation emails</label>
             <label index="flexforms.subject">Subject of subscription confirmation email</label>
             <label index="label.gender">Title</label>
```
Comment (bjo3rnf) [inline Resources/Private/Language/locallang.xml]: What about renaming this to ``label.title`` as 'gender' is not that appropriate here?

File: Resources/Private/Language/locallang.xml [hunk 2]
```diff
@@ -22,6 +22,8 @@
...
+            <label index="label.m">Mr.</label>
```
Comment (bjo3rnf) [inline Resources/Private/Language/locallang.xml]: ...and then make this read ``label.title.male`` and ``label.title.female``?

**Comment by bjo3rnf** [comment]
Hi Fabian,

thanks for your contribution. Please see my comments and reword your commit message to '[TASK] Make labels translatable', thanks. (Sorry for being picky here but I learned from the best ;) )

Cheers
Björn

**Comment by fabtho** [comment]
sorry, yes,  you pickiness sucks. ;-) why not just change the code yourself? but I will do as you requeste... wait some minutes..

**Comment by bjo3rnf** [comment]
I could have done that myself, yes. Sorry for not offering this to you ;)

**Comment by fabtho** [comment]
So

**Comment by fabtho** [comment]
Sorry, but I don't think this is a good Idea at all. Or it is a backwardscompatibily bracking change.  If somebody  overrides his template paths and use there own New.html changing the language key from label.gender to label.title will brack it. 

I will change label.title.male and label.title.female ok, but no label.gender ->  label.title. Then it will not bracking anything. 

What do you think?

**Comment by bjo3rnf** [comment]
You are right. Let's make a compromise: ``label.gender`` and ``label.gender.male``.

**Comment by cedricziel** [comment]
I think it's fine as long as it's just on master. Afterwards version numbers will help.

**Comment by bjo3rnf** [comment]
Manually picked and merged. Thanks.

**Comment by fabtho** [comment]
thanks too, more to come..

**Pull request closed without merging**